package.json

```diff
@@ -24,35 +24,35 @@
   ],
   "dependencies": {
     "hyphenate-style-name": "^1.0.0",
-    "lodash.omit": "^3.0.0",
-    "matchmedia": "^0.1.1",
-    "object-assign": "^2.0.0"
+    "matchmedia": "^0.1.2",
+    "object-assign": "^4.0.1"
   },
   "peerDependencies": {
-    "react": "^0.12.0 || ^0.13.0 || ^0.14.0"
+    "react": "^0.14.0"
```

Comment:
Somebody with react 0.12 might try the sample and conclude that everything's broken. I'd increase the version of react-responsive instead. I'll make a another PW if you prefer the old way.